src/analysis/normed_space/exponential.lean

```diff
@@ -117,6 +123,12 @@ begin
   simp [h]
 end
 
+end topological_algebra
+
+section any_field_any_algebra
+
```

Comment:
```suggestion
section normed_algebra

```
The old name looks a bit strange now that it's no longer the most general section in the file.

Reply:
There are multiple sections; how about:
```suggestion
section normed

section any_field_any_algebra

```

Reply:
Yes that sounds good too.